e2e/cypress/support/api_commands.js

```diff
@@ -661,18 +661,6 @@ Cypress.Commands.add('apiDeleteScheme', (schemeId) => {
     });
 });
 
-/**
- * Activate/Deactivate a User directly via API
- * @param {String} userId - The user ID
- * @param {Boolean} active - Whether to activate or deactivate - true/false
- */
-Cypress.Commands.add('apiActivateUser', (userId, active = true) => {
-    const baseUrl = Cypress.config('baseUrl');
-    const admin = getAdminAccount();
-
-    cy.externalRequest({user: admin, method: 'put', baseUrl, path: `users/${userId}/active`, data: {active}});
-});
```

Comment:
There is a reference of this function in the test guest_identification_ui_spec. Can you please update this test

Reply:
Thanks! I thought I changed it too but not. Now updated.